spyder/plugins/outlineexplorer/tests/test_widgets.py

```diff
@@ -298,120 +208,6 @@ def test_go_to_last_item(create_outlineexplorer, qtbot):
     assert outlineexplorer.treewidget.currentItem().text(0) == 'method1'
 
 
-@pytest.mark.skip(reason='Cell support is disabled temporarily')
-def test_code_cell_grouping(create_outlineexplorer):
```

Comment:
Is this test still failing? I think cell support is available, right? Seems like it could help checking the correct behavior of the Outline Explorer, maybe we should try to restore it instead of deleting it?

Reply:
This test worked for the previous implementation of the Outline (i.e. the one that didn't depend on the LSP), so it's not possible to restore it now.

However, I think there are some validations on it that are not performed in the new tests, so I'll try to port them.